src/python/pants/reporting/reporting.py

```diff
@@ -94,13 +94,6 @@ def register_options(cls, register):
             default=100.0,
             help="Rate at which to sample Zipkin traces. Value 0.0 - 100.0.",
         )
```

Comment:
I expect that there is a lot more code in here related to zipkin that came be removed.